nwbinspector/nwbinspector.py

```diff
@@ -1,290 +1,161 @@
+"""Cody Baker, Ben Dichter, and Ryan Ly."""
```

Comment:
Instead of listing authors in each Python module, I would suggest maybe adding a Contributors file to the repo. You may also want to update the list of authors in setup.py. I think docstrings of modules should ideally just briefly describe what the module does.

Reply:
Sure; I usually just do the authors by default to avoid `pydocstyle D100` warnings. Will swap them to brief descriptions, and add a contributors file.